Assets/Scripts/CopilotMechanics/MushroomFriendPassive.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MushroomFriendPassive : CopilotPassiveMechanic
+{
+    public float defenseIncrease; //amount to increase defense
+    public float sporeTimer; //lifetime of spores
+    private int healthLost; //set threshold amount of 10% health decrease
+    private int health; //to detect if health decreased or increased
+    void Start()
+    {
+        health = PlayerInfo.singleton.health;
+        healthLost = (int)(PlayerInfo.singleton.health * 0.1f);
```

Comment:
Same sort of thing for a previous comment. I would make a new instance variable for the 0.1f value just in case designers ever need to change it. Side note if you want you can default the value of the variable to 0.1f